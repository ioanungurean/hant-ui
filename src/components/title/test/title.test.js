import React, { Component } from 'react';
import Title from '../title.component';

describe('<Title />', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<Title title="Test Title" />);

    expect(wrapper).toMatchSnapshot();
  });
});
