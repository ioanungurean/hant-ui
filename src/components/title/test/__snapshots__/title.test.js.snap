// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render without throwing an error 1`] = `
<h1
  className="title"
>
  Test Title
</h1>
`;
